### ISSUE TITLE
Capturing updated slider value

### ISSUE THREAD

**Issue by pvijaysimha** [rdub80/aframe-gui#18]
I dont find a way to capture the updated percent of the slider

    <a-gui-slider id="slider"
            width="2.5" height="0.75"
            position="2 -2 0" scale="3 3 3" onclick="setVolume()"
            percent="0.29"
            margin="0 0 0.05 0"
    >
    </a-gui-slider>

What is the best way to do it?

**Comment by lmalave** [comment]
This fix was merged and was part of the 0.3.0 release.

**Comment by AliKarpuzoglu** [comment]
When I use  function(percent) I don't get the percentage, instead it outputs a "click" object without any 'percent' value

Edit: In case someone else has the same issue: 

the function should have 2 parameters, eg:
```

testSliderAction = function (click,percent) {
			console.log(percent);
		}
```

you can access the percent as the second parameter
